vignettes/metadata_analysis.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Metadata analysis using fulltext"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Metadata analysis using fulltext}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+In this vignette, we are going to demonstrate how the `fulltext` package can be used for meta analysis. The package is currently integrated with the following data sources:
```

Comment:
don't need to intorduce the package again, just focus on the topic of the meta-analysis